src/memo/memo.ml

```diff
@@ -178,15 +247,46 @@ module Cached_value = struct
         t.calculated_at <- Run.current ();
         Some t.data
     end
+  and get_sync : type a. a t -> a option = fun t ->
```

Comment:
I feel like `get_sync` should be defined in its own recursive group before `get_async`, given that `get_async` can call `get_sync` but the opposite is not allowed. What do you think?

Reply:
Yeah, definitely. This requires no other changes.